### ISSUE TITLE
`set-output` command deprecation

### ISSUE THREAD

**Issue by jcarvalho** [getsentry/action-release#108]
Hi.

As per https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/, the `set-command` output has been deprecated, which results in the following warning being issued whenever the action is used:

```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

The warning should go away by bumping `@actions/core` package to `1.10.0` or higher.

**Comment by armenzg** [comment]
Thank you @jcarvalho ! Added to our backlog.

**Comment by jcarvalho** [comment]
Can confirm this got fixed in 1.2.1 :+1: Thanks for tackling this!

**Comment by armenzg** [comment]
Thanks @jcarvalho !
